packages/client/src/forms/index.ts

```diff
@@ -546,6 +546,7 @@ export interface IInformativeRadioGroupFormField extends IFormFieldBase {
 export interface ITextFormField extends IFormFieldBase {
   type: typeof TEXT
   maxLength?: number
+  dynamicOptions?: IDynamicOptions
```

Comment:
```suggestion
  dependency?: string
```
Wouldn't it make more sense this way?

Reply:
`resetDependentSelectValues` uses the `IDynamicOptions` structure for resting the value.

Reply:
Fair enough. It's just that it's kinda out of context for `dynamicOptions` to be present for a _Text Field_. We can probably use something along the line of
```ts
(field.type === SELECT_WITH_DYNAMIC_OPTIONS && field.dynamicOptions.dependency === fieldName) ||
(field.type === TEXT && field.dependency === fieldName)
```